requirements.txt

```diff
@@ -0,0 +1,14 @@
+dask==0.17.2
+dask-ml==0.4.1
+distributed==1.21.4
+numpy
+pandas
+scikit-learn
+matplotlib
+h5py
+graphviz
+nbconvert
+nbformat
+pathlib
+jupyter_client
+ipykernel
```

Comment:
We currently keep an environment.yml file that we use with the Binder deployment.  I wonder if we can reuse that here instead?  I worry a bit about having to keep the two in sync.  

I tend to use miniconda for testing.  I'm not sure if we want to do that here, but if so I recommend stealing the following lines: https://github.com/mrocklin/streamz/blob/c180551ff5a7eb7aa30bf56242554144f726d45a/.travis.yml#L17-L22

and then following up with 

```
conda env create -f environment.yaml -n test
source activate test
```

Reply:
I will investigate using miniconda instead